code/parse/sexp.h

```diff
@@ -808,6 +808,8 @@ struct ship_obj;
 #define OP_CONTAINER_REMOVE_FROM_MAP		(0x004b | OP_CATEGORY_CHANGE2 | OP_NONCAMPAIGN_FLAG)	// Karajorma/jg18
 #define OP_CONTAINER_GET_MAP_KEYS			(0x004c | OP_CATEGORY_CHANGE2 | OP_NONCAMPAIGN_FLAG)	// Karajorma/jg18
 #define OP_CLEAR_CONTAINER					(0x004d | OP_CATEGORY_CHANGE2 | OP_NONCAMPAIGN_FLAG)	// Karajorma/jg18
+#define OP_ADD_BACKGROUND_BITMAP2			(0x004e | OP_CATEGORY_CHANGE2 | OP_NONCAMPAIGN_FLAG)	// Goober5000
+#define OP_ADD_SUN_BITMAP2					(0x004f | OP_CATEGORY_CHANGE2 | OP_NONCAMPAIGN_FLAG)	// Goober5000
```

Comment:
I have a slight preference for a SEXP name like `add-sun-bitmap-new` or some other name to indicate that this SEXPs should be used instead of `add-sun-bitmap`, and similar for backgrounds.

A name like `add-sun-bitmap2` could just mean a variant of the original, rather than the replacement, even if the original is marked in the SEXP help text as deprecated.

Reply:
Changed.  Also changed `add-to-colgroup2` and `remove-from-colgroup2` since they follow the same pattern.